src/components/ProFullCourt/index.tsx

```diff
@@ -32,34 +64,41 @@ const ProFullCourt = () => {
         {({ store }) => (
           <Stage
             id="basketball-court"
-            width={850}
-            height={520}
+            height={stageHeight}
+            width={stageWidth}
+            scaleX={courtRatio}
+            scaleY={courtRatio}
             visible={true}
             style={{ backgroundColor: "white" }}
+            data-testid="stage"
           >
             <Provider store={store}>
               <Layer>
                 {/* border only for pro full court size */}
-                <Border courtRatio={courtRatio} />
+                <Border startPoint={startPoint} />

```

Comment:
"startPoint={startPoint}" is passed to each child component. Is it possible to use React useContext hook here and the following components?

Reply:
yes, it works~~~~~updated~~Cheers~~